packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 51.7.0 - Mar 30, 2021
+
+## Added
+
+#### salesforcedx-sobjects-faux-generator
+
+- Generate typings for sobjects ([PR #3018](https://github.com/forcedotcom/salesforcedx-vscode/pull/3018))
+
```

Comment:
FYI - pending https://github.com/forcedotcom/salesforcedx-vscode/pull/3101, which I should have resolved today - we should have an entry here for...

#### salesforcedx-vscode-core

- Switched setting 'salesforcedx-vscode-core.experimental.deployRetrieve' to on by default. ([PR #3101](https://github.com/forcedotcom/salesforcedx-vscode/pull/3101))

CC @brpowell in case you'd prefer a different wording.